resolver/resolver.go

```diff
@@ -586,6 +591,23 @@ func (r *Resolver) lookupPackageNameMapUsedInGRPCFederationDefinitionFromMessage
 	return pkgNameMap
 }
 
+func (r *Resolver) lookupPackageNameMapFromGRPCError(err *GRPCError) map[string]struct{} {
+	if err == nil {
+		return nil
+	}
+	pkgNameMap := map[string]struct{}{}
+	maps.Copy(pkgNameMap, r.lookupPackageNameMapFromCELValue(err.If))
+	maps.Copy(pkgNameMap, r.lookupPackageNameMapFromCELValue(err.Message))
+	maps.Copy(pkgNameMap, r.lookupPackageNameMapFromCELValue(err.IgnoreAndResponse))
+	for _, detail := range err.Details {
+		maps.Copy(pkgNameMap, r.lookupPackageNameMapFromCELValue(detail.If))
```

Comment:
Other fields ( `DefSet` `Messages` `PreconditionFailures` `BadRequests` `LocalizedMessages` ) in `GRPCErrorDetail` may also need to be covered.